src/pages/en/editor-setup.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import Badge from '~/components/Badge.astro';
+title: Editor Setup
+description: Set up your editor to build with Astro.
+---
+You can build your Astro site with any code editor, installed locally or in the cloud!
+
+This page explains how to improve the experience of editing `.astro` files in supported editors.
+
+## VS Code Extension
+
+[VS Code](https://code.visualstudio.com) is a popular code editor that can be dowloaded locally, and is also used by online platforms such as [GitHub Codespaces](https://github.com/features/codespaces), and [Gitpod](https://gitpod.io). 
+
+Astro has its own [Astro VS Code Extension on Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) that you can install in any of these VS Code environments for .`astro` syntax highligting, code completion, and more!
+
+### Full List of Features
+- [Go to Definition](https://code.visualstudio.com/docs/editor/editingevolved#_go-to-definition)
+- Code hover hints 
+- Code completion
+- Function signatures
+- Syntax highlighting
+- Code folding
+- Emmet
+
+### Installation
+
+
+After [setting up VS Code](https://code.visualstudio.com/Docs/setup/setup-overview) and running any of our [Astro starter templates](https://github.com/withastro/astro/tree/main/examples), the Astro VS Code extension should be recommended to you automatically by a pop-up (in the bottom right corner).
+
+To add the extension yourself:
+
+1. Navigate to the "Extensions" tab ("Ctrl-Shift-X")
```

Comment:
If we want to include the keyboard shortcut it should probably include the Mac variation (Cmd+Shift+X).

Also, there’s a special `<kbd>` element for keyboard shortcuts if you fancy using it:

```html
<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd>
```

Renders:

<kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>X</kbd>

(Not sure if the docs site has styling in place for `<kbd>`.)

Reply:
Doesn't appear to...  :(  Maybe we need to bug @bholmesdev about this....

![Screenshot 2022-03-29 17 53 01](https://user-images.githubusercontent.com/5098874/160705194-f89670ef-cc01-4990-adbc-02caf6662ed0.png)

So, maybe for both reasons (no fancy styling, needing to type multiple variations), I'll just take it out. Another site gave the keyboard shortcuts for their install instructions, and I thought that was kind of neat as visual variety. But I don't want to make it a Whole. Thing.

Reply:
You can also link to this docs page on extensions! Should keep our instructions evergreen too: https://code.visualstudio.com/docs/editor/extension-marketplace#_browse-for-extensions

Reply:
I’d happily add some custom styles, but I also think the keyboard shortcut may be superfluous in this case.